partcad/src/partcad/part_factory_brep.py

```
@@ -0,0 +1,134 @@
+        """
+        Process a BREP file using a subprocess for larger files.
+        """
+        # Initialize runtime if not already done
+        if self.runtime is None:
+            pc_logging.debug("Initializing subprocess runtime...")
+            self.runtime = self.ctx.get_python_runtime(self.PYTHON_RUNTIME_VERSION)
+            if self.runtime is None:
+                raise RuntimeError("Failed to initialize runtime for subprocess execution.")
+            pc_logging.debug(f"Subprocess runtime initialized: {self.runtime}")
+
+        wrapper_path = wrapper.get("brep.py")
```

Ruff (0.8.2)

Undefined name `wrapper`

(F821)